src/Middleware/HttpLogging/src/W3CLoggerOptions.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+
+namespace Microsoft.AspNetCore.HttpLogging
+{
+    /// <summary>
+    /// Options for the <see cref="W3CLogger"/>.
+    /// </summary>
+    public class W3CLoggerOptions
+    {
+        private int? _fileSizeLimit = 10 * 1024 * 1024;
+        private string _fileName = "serverlog-";
+        private string _logDirectory = "C:\\code\\scratch\\W3CLogs";
```

Comment:
`Path.Join(IHostEnvironment.ContentRootPath , "logs")`

Reply:
Do I get the IHostEnvironment thru DI?

Reply:
Yes